arviz/tests/base_tests/test_utils.py

```diff
@@ -291,3 +291,48 @@ def test_normalize_angle(mean):
 
     values = _normalize_angle(rvs, zero_centered=False)
     assert ((values >= 0) & (values <= 2 * np.pi)).all()
+
+
+@pytest.mark.parametrize("mean", [[0, 0], [1, 1]])
+@pytest.mark.parametrize(
+    "cov",
+    [
+        np.diag([1, 1]),
+        np.diag([0.5, 0.5]),
+        np.diag([0.25, 1]),
+        np.array([[0.4, 0.2], [0.2, 0.8]]),
+    ],
+)
+@pytest.mark.parametrize("contour_sigma", [np.array([1, 2, 3])])
+def test_find_hdi_contours(mean, cov, contour_sigma):
```

Comment:
I'm a bit worried that this unit test is overly complicated, but I wasn't sure how best to approach it.

Reply:
I think is ok.